OperatorOverload.R

```diff
@@ -0,0 +1,52 @@
+# Used for guards
+OperatorOverload <- T
+
+if (!exists('CoreVarFile'))
+    source('CoreVariable.R')
+
+# Habar n-am daca are sens sau e o aberatie
+setMethod(f="+",
+    signature=c("CoreVariable", "CoreVariable"),
+    definition=function(e1, e2)
+    {
+        new_pmf <- function(x) {
+            convolutie <- function(k) {
+                return(e1@pmf(k) * e2@pmf(x - k))
+            }
+            return(integrate(convolutie, -Inf, Inf)$value)
+        }
+
+        new_var <- BuildFromPMF(new_pmf)
+        return(new_var)
+    }
+)
+
+## WHAT IS BELLOW ONLY EXISTS FOR DEBUGGING.
+## THIS MODULE SHOULD NOT CONTAIN ANY VARIABLES OR ADDITIONAL SOURCES
+
+# CDF and PMF constructors of the CoreVariable.
+if (!exists('ConstructorsFile'))
+    source('Constructors.R')
+
+    
+std <- function(x) {
+  return(exp(-x*x/2)/sqrt(2*pi))
```

Comment:
asta e un exemplu? sau care e faza, de ce se afla in operator overload. nu inteleg ce vrei sa faci aici

Reply:
ah, acum vad, vrei sa testezi mai jos la linia 45, 46 si 48 daca merge ce ai facut